src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -51,7 +64,7 @@ export const jContentAccordionItems = registry => {
         getPath,
         getUrlPathPart,
         getPathForItem: node => {
-            return node.ancestors[node.ancestors.length - 1].path;
+            return node.ancestors.sort(compareNodesByPathDesc)[node.ancestors.length - 1].path;
```

Comment:
why do you have to sort ancestors ? they are always anyway sorted by path length (maybe in the opposite order ?)

Reply:
yes, right! they are actually sorted